util/filter_bench.cc

```diff
@@ -378,7 +381,7 @@ void FilterBench::Go() {
       info.filter_ = info.plain_table_bloom_->GetRawData();
     } else {
       if (!builder) {
-        builder.reset(&dynamic_cast<BuiltinFilterBitsBuilder &>(*GetBuilder()));
+        builder.reset(static_cast_with_check<BuiltinFilterBitsBuilder, FilterBitsBuilder>(GetBuilder()));
```

Comment:
The second template argument should not be needed, unless some compiler needs it (which apparently we would not be compiling db_flush_test.cc with).